src/plugins/sails.js

```diff
@@ -169,7 +169,7 @@ io.socket.on('gameCreated', function (evData) {
 });
 
 io.socket.on('gameStarting', function (evData) {
-  store.commit('removeGame', evData);
+  store.commit('gameStarted', evData);
```

Comment:
Changed the name of this mutation because it's no longer removing a game from the list, now it just moves the game over to a different list for the spectatable games